readme.md

```diff
@@ -150,7 +150,9 @@ If you want to contribute, please read the [contribution guidelines](contributin
 
 ### Libraries
 
-- [GraphQL](https://github.com/apollostack/eslint-plugin-graphql) - Check your GraphQL query strings against a schema.
+- GraphQL
+  - [`dotansimha/graphql-eslint`](https://github.com/dotansimha/graphql-eslint) - Validates, prettifies and checks your GraphQL operations and GraphQL schema for best-practices.
```

Comment:
Can you remove the backticks on both of these, please?

Reply:
Done!